habitat_baselines/il/models/models.py

```diff
@@ -9,6 +9,8 @@
 
 from habitat import logger
 
+HiddenState = Union[Tensor, Tuple[Tensor, Tensor], None]
```

Comment:
The state of the RNN can be nothing, a tensor (GRU) or two tensors (LSTM). Added a new type to reflect that. Name is debatable.

Reply:
Alternatively `Optional[Union[Tensor, Tuple[Tensor, Tensor]]]`